compiler.pri

```diff
@@ -248,10 +248,10 @@ unix {
 		}
 	} else {
 		# If C++11 support hasn't been explicitly enabled,
-		# force C++03 mode. If we don't do this, newer
+		# force C++98/C++03 mode. If we don't do this, newer
```

Comment:
Well, it's not really a "workaround". For GCC, `-std=c++98` == `-std=c++03` == `-ansi`. They mean the same thing (according to the Debian man page, at least). It's just that GCC 4.1 doesn't have the `-std=c++03` alias, so we have to do this for older stuff.